options/components/webTopContent/top-contents/WebGlobalNavigationBar.jsx

```diff
@@ -0,0 +1,50 @@
+import { useContext } from "react";
+
+import { WebSearchContext } from "../../../context/WebSearchContext";
+
+export default function WebGlobalNavigationBar({ urlNewList }) {
+  const { searchResultCount } = useContext(WebSearchContext);
+
+  return (
+    <div className="mb-5 my-4">
+      <h1 className="font-bold text-lg">Web View</h1>
+      {searchResultCount ? (
+        <UserSearchBookmarkView />
+      ) : (
+        <UserBookmarkListView urlNewList={urlNewList} />
+      )}
```

Comment:
```suggestion
      {searchResultCount ?
        <UserSearchBookmarkView /> :
        <UserBookmarkListView urlNewList={urlNewList} />
      }
```
다른 JSX를 컴포넌트화 시켜서 깔끔한 삼항연산자로 보이는 이 부분은 좋은것 같습니다.
제 생각에는 searchResultCount 변수명을 수정해야 할것 같습니다. Count가 포함되어 있어 숫자형을 표현하는 변수로 해석이 되고, 삼항연산자에서 소괄호는 제외해도 될 것 같습니다.

Reply:
사용자가 검색 시 그 이후인 결과를 보여주는 UI이며 갯수를 보여준다는 생각으로 count를 사용하였으나 말씀 주신 부분 공감하여 아래에 주신 피드백 참조해 `hasSearchResult` 를    `setHasSearchResult`로 변경하겠습니다.